pkgs/applications/graphics/foxotron/default.nix

```diff
@@ -67,7 +67,7 @@ stdenv.mkDerivation rec {
       Revision 2021 3D Graphics Competition.
     '';
     homepage = "https://github.com/Gargaj/Foxotron";
-    license = licenses.publicDomain;
+    license = licenses.unlicense;
```

Comment:
Didn't notice [this link](https://github.com/Gargaj/Foxotron/blob/2021-04-19/LICENSE#L28) when I packaged it, everything else just refers to it as "public domain".